src/lib/bcmp/bcmp.cpp

```diff
@@ -182,7 +183,12 @@ int32_t bmcp_process_packet(struct pbuf *pbuf, ip_addr_t *src, ip_addr_t *dst) {
       case BCMP_CONFIG_STATUS_RESPONSE:
       case BCMP_CONFIG_DELETE_REQUEST:
       case BCMP_CONFIG_DELETE_RESPONSE: {
-        bcmp_process_config_message(static_cast<bcmp_message_type_t>(header->type), header->payload);
+        bool should_forward = bcmp_process_config_message(
+            static_cast<bcmp_message_type_t>(header->type), header->payload);
+        if (should_forward) {
+          // Forward the message to all ports other than the ingress port.
+          bcmp_ll_forward(pbuf, ingress_port);
+        }
```

Comment:
If the message can't be processed at this node, forward it, calling the new `bcmp_ll_forward` function.

Reply:
I wonder if we need to do this for all places we have `multicast_ll_addr`

I see references currently in: 
`bcmp_heartbeat`
`bcmp_neighbors`
`bcmp_resource_discovery`
`bcmp_time` 

Maybe doesn't need to be this PR but assuming that they suffer from the same behavior?

Reply:
Actually I don't think heartbeat or neighbors need to change?
But time and resource_discovery likely have the same issue.

Reply:
Good call. I'll create a new story for resource discovery and time now and can code them up today actually in a separate PR.